docs/src/main/sphinx/security/file-system-access-control.md

```diff
@@ -490,6 +491,15 @@ connector](/connector/delta-lake). It allows all users to execute the
 }
 ```
 
+(system-file-table-procedure-rules)=
+#### Table procedure rules
+
+Table procedures are executed using the
+[ALTER TABLE ... EXECUTE](/sql/alter-table#alter-table-execute) syntax.
```

Comment:
Hopefully this is the right way to link this

Reply:
nope .. you have to introduce an anchor there if you want to link to a specific section..

Reply:
Eh, not a power-user of MD, so I though that `(alter-table-execute)=` thingy *was* the anchor :)

> [ALTER TABLE ... EXECUTE](alter-table-execute)

But it's in a different file, so how does that work?

Reply:
Hahah .. reference anchors (and function anchors and so on) in sphinx are global and independent of the filename .. they are all collected before rendering out HTML .. verified and then adjusted to whatever file the reference points to

Reply:
This is a sphinx thing .. not md .. we just use the myst parser with sphinx .. and myst parser is a markdown extension for various systems.. hth